.github/workflows/scala3-compile.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         command:
-          - akka-testkit/Test/compile akka-actor-tests/Test/compile
+          - akka-testkit/Test/compile akka-actor-tests/Test/compile akka-osgi/Test/compile
```

Comment:
Could we instead not publish akka-osgi for Scala 3? If possible no need to compile and test at all for Scala 3.

Reply:
I agree that would be nicer, progress on that in #30981